content/docs/add-react-to-an-existing-app.md

```diff
@@ -6,91 +6,92 @@ prev: add-react-to-a-new-app.html
 next: cdn-links.html
 ---
 
-You don't need to rewrite your app to start using React.
+Use as little, or as much of React as you need.
 
-We recommend adding React to a small part of your application, such as an individual widget, so you can see if it works well for your use case.
+While you could [create a single-page app](/docs/add-react-to-a-new-app.html) with React, **the vast majority of the websites aren't, and don't need to be single-page apps**.
 
-While React [can be used](/docs/react-without-es6.html) without a build pipeline, we recommend setting it up so you can be more productive. A modern build pipeline typically consists of:
+React has always been designed for progressive adoption, and **you can use as little or as much of React as you need**. Quite often, people only want to add some "sprinkles of interactivity" to an existing page, and React components are a great tool for that.
 
-* A **package manager**, such as [Yarn](https://yarnpkg.com/) or [npm](https://www.npmjs.com/). It lets you take advantage of a vast ecosystem of third-party packages, and easily install or update them.
-* A **bundler**, such as [webpack](https://webpack.js.org/) or [Browserify](http://browserify.org/). It lets you write modular code and bundle it together into small packages to optimize load time.
-* A **compiler** such as [Babel](http://babeljs.io/). It lets you write modern JavaScript code that still works in older browsers.
+In some cases, [more parts of the page become driven by React over time](https://www.youtube.com/watch?v=BF58ZJ1ZQxY), but in others React stays as a flexible and unopinionated tool alongside the existing markup and code.
 
-### Installing React
+### 1. Add React [CDN links](/docs/cdn-links.html) to your HTML
 
->**Note:**
->
->Once installed, we strongly recommend setting up a [production build process](/docs/optimizing-performance.html#use-the-production-build) to ensure you're using the fast version of React in production.
-
-We recommend using [Yarn](https://yarnpkg.com/) or [npm](https://www.npmjs.com/) for managing front-end dependencies. If you're new to package managers, the [Yarn documentation](https://yarnpkg.com/en/docs/getting-started) is a good place to get started.
-
-To install React with Yarn, run:
-
-```bash
-yarn init
-yarn add react react-dom
+```html
+<script
+  crossorigin
+  src="https://unpkg.com/react@16/umd/react.development.js">
+</script>
+<script
+  crossorigin
+  src="https://unpkg.com/react-dom@16/umd/react-dom.development.js">
+</script>
 ```
 
-To install React with npm, run:
+### 2. Write your components
 
-```bash
-npm init
-npm install --save react react-dom
+```js
+// src/button.js
+
+class Button extends React.Component {
```

Comment:
prefer to use functional components unless you have a need for classes

Reply:
This is familiar syntax that always works. Shouldn't have to explain function components on the install page.